src/py/aspen/database/models/sequences.py

```diff
@@ -146,7 +146,7 @@ class PathogenGenome(Entity):
     __tablename__ = "pathogen_genomes"
 
     entity_id = Column(Integer, ForeignKey(Entity.id), primary_key=True)
-    sequence = Column(String, nullable=False)
+    sequence = deferred(Column(String, nullable=False))
```

Comment:
so this is storing the actual sequence data?? i thought we would be storing an s3 key or something like that

Reply:
no, we're storing the sequence data in the db.